src/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -172,6 +172,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(smoke_ConversionLayerTest/ConversionLayerTest.CompareWithRefs.*UNSPECIFIED.*)",
         // Issue:
         R"(.*smoke_VariadicSplit4D_CPU_zero_dims.*)",
+        // New API tensor tests
+        R"(.*OVInferRequestCheckTensorPrecision.*)",
```

Comment:
how is it different from https://github.com/openvinotoolkit/openvino/blob/434d2436dc6f2d92b142f3d50abe47ce680acaf3/src/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp#L165-L172 ?

Reply:
ConversionLayerTests checks Convert ot ConvertLike operations.